locale/en/blog/release/v16.5.0.md

```diff
@@ -0,0 +1,247 @@
+---
+date: 2021-07-14T09:12:10.881Z
+version: 16.5.0
+category: release
+title: Node v16.5.0 (Current)
+slug: node-v16-5-0
+layout: blog-post.hbs
+author: Michaël Zasso
+---
+
+### Notable Changes
+
+#### Experimental Web Streams API
+
+Node.js now exposes an experimental implementation of the
+[Web Streams API](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API).
+
+While it is experimental, the API is not exposed on the global object and is only
+accessible using the new `stream/web` core module:
+
+```mjs
+import { ReadableStream, WritableStream } from 'stream/web';
+// Or from 'node:stream/web'
+```
```

Comment:
Anybody know why the single quotes are being converted to `&apos` and displaying incorrectly on the website?

![image](https://user-images.githubusercontent.com/718899/125605701-9073df9d-2851-49bf-a7c0-2321b764b8f2.png)

Reply:
https://github.com/nodejs/nodejs.org/issues/3976 fixed by https://github.com/nodejs/nodejs.org/pull/3977